ui/analyse/src/ctrl.ts

```diff
@@ -785,7 +785,7 @@ export default class AnalyseCtrl {
       !isTouchDevice() &&
       !chap?.practice &&
       chap?.conceal === undefined &&
-      !chap?.gamebook &&
+      !(chap?.gamebook && !this.study?.members.canContribute()) &&
```

Comment:
but then are they displayed to the person *playing* the gamebook?

Reply:
Wouldn't a contributor always get the editor UI?

Reply:
a non-contributor playing? no. but yes on contributor previews which is probably not ideal, so fixed in 0a93bcf